packages/nextra-theme-docs/src/mdx-components.tsx

```diff
@@ -184,7 +183,7 @@ const Summary = (props: ComponentProps<'summary'>): ReactElement => {
       className={cn(
         'list-none cursor-pointer rounded p-1 outline-none transition-colors hover:bg-gray-100 dark:hover:bg-neutral-800',
         "before:content-[''] before:inline-block before:transition-transform dark:before:invert",
-        '[[data-open]>&]:before:rotate-90'
+        '[[open]>&]:before:rotate-90 rtl:before:rotate-180'
```

Comment:
to fix summary icon (should be inverted)
<img width="245" alt="image" src="https://user-images.githubusercontent.com/7361780/184453381-71a0e41d-03e1-41ac-822b-10abf7eb9594.png">